[program:bugtest]

command=python3 /home/fy/code/supervisor-py3-bug/test.py
directory=/home/fy/code/supervisor-py3-bug

process_name=%(program_name)s_%(process_num)1d
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/supervisor/lb-%(process_num)1d.log
stdout_logfile_maxbytes=1024MB
stdout_logfile_backups=512
stdout_capture_maxbytes=1MB
stdout_events_enabled=false
loglevel=warn
numprocs_start=1
numprocs=1

